playwright_stealth/stealth.py

```diff
@@ -149,6 +150,35 @@ async def stealth_async(page: AsyncPage, config: StealthConfig = None):
     for script in (config or StealthConfig()).enabled_scripts:
         scripts.append(script)
 
-    combined_script = "\n".join(scripts)
+    add_stealth_headers(page)
+
+    combined_script = "\n".join(scripts) + "\nconsole.log(`end`);"
 
     await page.add_init_script(combined_script)
+
+
+async def add_stealth_headers(page: AsyncPage):
+    """Adds stealth headers to the page"""
+    page.route(
+        "**/*",
+        lambda route, request: route.continue_(
+            headers={
+                **request.headers,
+                "sec-ch-ua": '"Chromium";v="129", "Not=A?Brand";v="8"',
```

Comment:
curious what the rationale is for setting a full explicit string vs removing keys via regex?  iiuc if we lock our fingerprint in to something very specific as is here, this can also over time be an indication to sites that this exact fingerprint is a bot, vs leveraging existing headers and only modifying the giveaway signals means there will still be variance

Reply:
I'm still experimenting on this but there is still some header & client side inconsistencies for some reason. I need more time on researching which combinations and attributes will result in greater consistency.